java/org/apache/catalina/storeconfig/ConnectorStoreAppender.java

```diff
@@ -29,6 +29,7 @@
 import java.util.List;
 import java.util.Set;
 
+import org.apache.catalina.Globals;
```

Comment:
I see that we also have a file named `java/org/apache/catalina/storeconfig/Constants.java` but it does not set `CATALINA_BASE_PROP`.  I am guessing there is a reasoning behind the different `Constants` classes so trying to figure it out.